examples/fabric/v4/cloudRouterConnectivity/routing-protocol-direct/README.md

```diff
@@ -1,29 +1,25 @@
 # ECX Fabric Cloud Router Connection RP CRUD operations
-This example shows how to create Config Direct RP on FCR connection .
+This example shows how to Config Direct Routing Protocol details for an existing FCR connection .
 
-Note: Each time you need to create a RP resource add-on
-make a copy of the base folder - examples/routing-protocol-direct/ and CD into this folder to perform all the CRUD operations.
+Note: Each time you need to create a Direct RP resource add-on
+make a copy of the base folder - examples/fabric/v4/cloudRouterConnectivity/routing-protocol-direct/ and CD into this folder to perform all the CRUD operations.
 
 ## Define values for the Fabric Cloud Router create
 At minimum, you must set below variables in `terraform.tfvars` file:
-- `equinix_client_id` - Equinix client ID (consumer key), obtained after
-  registering app in the developer platform
-- `equinix_client_secret` - Equinix client secret ID (consumer secret),
-  obtained same way as above
+- `equinix_client_id` - Equinix client ID (consumer key), obtained after registering app in the developer platform
+- `equinix_client_secret` - Equinix client secret ID (consumer secret),obtained same way as above
 - `rp_name`- Name of routing Protocol
 - `rp_type`- Type of routing Protocol entity, "DIRECT" or "BGP"
-- connection_uuid = "d557cb4c-9052-4298-b5ca-8a9ed914cf03"
-  rp_type = "DIRECT"
-  rp_name = "FCR-RP"
-  equinix_ipv4_ip = "192.1.1.1/30"
-  equinix_ipv6_ip = "192::1:1/126"
+- `connection_uuid` = FCR Connection UUID
+- `equinix_ipv4_ip` = Equinix Side IpV4 Address
+- `equinix_ipv6_ip` = Equinix Side IpV6 Address
 
 
 ## Initialize
 - First step is to initialize the terraform directory/resource we are going to work on.
-  In the given example, the folder to perform CRUD operations on an RP resource can be found at examples/routing-protocol-direct/.
+  In the given example, the folder to perform CRUD operations on an RP resource can be found at examples/fabric/v4/cloudRouterConnectivity/routing-protocol-direct/.
 
-- Change directory into - `CD examples/routing-protocol-direct/`
+- Change directory into - `CD examples/fabric/v4/cloudRouterConnectivity/routing-protocol-direct/`
 - Initialize Terraform plugins - `terraform init`
 
 ## Routing-protocol Direct IP : Create, Read, Update and Delete(CRUD) operations
```

Comment:
In the table below (not shown because the diff isn't here) we should be specifying that we are creating a Direct Routing Protocol resource. Not FCR as stated.